src/cli/README_BR.md

```diff
@@ -235,6 +236,28 @@ prefix:1200:abba:baba:0::/64, on-link:yes, ms-since-rx:29527, lifetime:1800, pre
 Done
 ```
 
+### raoptions
+
+Usage: `br raoptions <options>`
+
+Sets vendor-specific options to append at the end of emitted Router Advertisement (RA) messages. `<options>` provided as hex bytes.
```

Comment:
nit: avoid possible confusion with "vendor-specific" since these options could be standard options defined in RFCs.
```suggestion
Sets additional options to append at the end of emitted Router Advertisement (RA) messages. `<options>` provided as hex bytes.
```

Reply:
Good idea. Thanks. Will change in next push.